src/lib/non_zero_curve_point/non_zero_curve_point.ml

```diff
@@ -215,7 +232,7 @@ module Stable = struct
 end
 
 (* bin_io omitted *)
```

Comment:
nit but if the other comments changed to be `bin_io, sexp omitted`, shouldn't this one too?

Reply:
Yes, well, I already merged before noticing this comment. There are other comments mentioning that the sexp functions are manually-written, so I'm not too worried.